kumascript/src/errors.js

```diff
@@ -223,6 +223,13 @@ class MacroBrokenLinkError extends MacroExecutionError {
   }
 }
 
+class MacroWrongXRefError extends MacroBrokenLinkError {
+  constructor(error, source, token) {
+    super(error, source, token, false);
+    this.name = "MacroWrongXRefError";
+  }
+}
+
```

Comment:
Nit: The `false` argument to `super()` is ignored since it's calling the constructor for `MacroBrokenLinkError` not `MacroExecutionError`.

```suggestion
class MacroWrongXRefError extends MacroBrokenLinkError {
  constructor(error, source, token) {
    super(error, source, token);
    this.name = "MacroWrongXRefError";
  }
}
```